core/src/modules/ops/profile.ts

```diff
@@ -222,7 +222,8 @@ export namespace ProfileOps {
     profileProperties: {
       [key: string]: Array<string | number | boolean | Date> | any;
     }[],
-    toLock = true
+    toLock = true,
+    ignoreMissingProperties = false
```

Comment:
What's the case where we want `ignoreMissingProperties=true`?

Reply:
This is used in the associateProfiles task. If the import data has a property that no longer exists, we want to just ignore it instead of throwing an error.